test/distributed_passes/auto_parallel_pass_test_base.py

```diff
@@ -37,7 +37,7 @@ class AutoParallelPassTestBase(DistPassTestBase):
     def setUp(self):
         paddle.enable_static()
         seed = int(os.environ.get('SEED', -1))
-        os.environ["FLAGS_dynamic_static_unified_comm"] = "0"
+        os.environ["FLAGS_dynamic_static_unified_comm"] = "1"
```

Comment:
=1 is no need since it is true by default

Reply:
I will delete the redundant code. Thx